content/en/dashboards/ddsql_editor/reference/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: DDSQL Reference
+---
+
+{{< callout url="https://google.com">}}
+DDSQL is in private beta.
+{{< /callout >}}
+
+## Overview
+
+DDSQL is a query language for Datadog data. It implements several standard SQL functions, such as `SELECT`, along with new functions for working with Datadog tags, metrics, and so on. For example, [the `AGGR` statement][5], which aggregates metrics data, is unique to DDSQL.
+
+You can use DDSQL to query your infrastructure, RUM data, and metrics in the [DDSQL Editor][2] in Datadog.
+
+DDSQL uses a [schema on read](#schema-on-read) approach to allow queries against unstructured data, such as [tags][1]. Each query is executed within a [session](#sessions) that can be configured with environment variables.
+
+## Supported SQL syntax
+
+SQL is broken into five different categories of statements. The table below indicates which categories are supported by DDSQL.
+
+| Category   | Examples    | Support      |
+|------------|-------------|--------------|
+| DQL (Data Query Language)   | `SELECT`, `AGGR` (DDSQL alternative to `SELECT`) | Supported    |
+| DML (Data Modification Language)  | `INSERT`, `UPDATE`, `DELETE`   | Limited: Data is not persisted across sessions |
+| DDL (Data Description Language)  | `CREATE`   | Limited: Data is not persisted across sessions |
+| DCL (Data Control Language)        | `GRANT`, `REVOKE`   | Not supported       |
+| TCL (Transaction Control Language) | `BEGIN`, `END`, `ROLLBACK`  | Not supported     |
```

Comment:
I don't think we should include this.

Reply:
@joparisot The other documentation (query templates, function signatures, etc.) refers to DQL_expression as a placeholder and DQLExpr as a type, and those link here. So I think we do need to keep it for the docs to make sense, but I agree that it's awkward. Are you okay with those adjustments being a future revision, since we're down to the wire with DASH?

Reply:
that works with me